modules/light-clients/09-localhost/client_state.go

```diff
@@ -108,13 +110,14 @@ func (cs ClientState) VerifyMembership(
 
 // VerifyNonMembership is a generic proof verification method which verifies the absence of a given CommitmentPath at a specified height.
 // The caller is expected to construct the full CommitmentPath from a CommitmentPrefix and a standardized path (as defined in ICS 24).
+// Note the store provided to the localhost client is the full IBC core state store.
```

Comment:
similar to what I did for `VerifyMembership`?

Reply:
Updated :)